### PR TITLE
Updating the NuGet package id to correct format

### PR DESCRIPTION
**Pull request by soninaren**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-version: 2.0.0-{build}
+version: 2.0.{build}
 pull_requests:
-  do_not_increment_build_number: true
+  do_not_increment_build_number: false
```
Comment (fabiocav) [inline appveyor.yml]: Why the change?

File: Functions.Templates/ProjectTemplates.nuspec [hunk 2]
```diff
@@ -1,13 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
-    <id>Microsoft.AzureFunctions.ProjectTemplates</id>
+    <id>Microsoft.Azure.Functions.ProjectTemplates</id>
```
Comment (fabiocav) [inline Functions.Templates/ProjectTemplates.nuspec]: This may have to use the existing `Microsoft.Azure.WebJobs.*` namespace

File: Functions.Templates/ProjectTemplates.nuspec [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
-    <id>Microsoft.AzureFunctions.ProjectTemplates</id>
+    <id>Microsoft.Azure.Functions.ProjectTemplates</id>
     <version>1.0.0</version>
     <authors>Microsoft</authors>
     <description>Azure Functions Project Template Pack for Microsoft Template Engine</description>
     <language>en-US</language>
     <projectUrl>https://go.microsoft.com/fwlink/?linkid=848145</projectUrl>
     <licenseUrl>https://raw.githubusercontent.com/Azure/azure-webjobs-sdk-templates/8ba7a6f1f907edb5de30c1ef4791d97711d84cb6/LICENSE.txt</licenseUrl>
+    <copyright>© Microsoft Corporation. All rights reserved.</copyright>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
```
Comment (fabiocav) [inline Functions.Templates/ProjectTemplates.nuspec]: Use the aka.ms link used by the other projects here.

File: Functions.Templates/ProjectTemplates.nuspec [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
-    <id>Microsoft.AzureFunctions.ProjectTemplates</id>
+    <id>Microsoft.Azure.Functions.ProjectTemplates</id>
     <version>1.0.0</version>
     <authors>Microsoft</authors>
     <description>Azure Functions Project Template Pack for Microsoft Template Engine</description>
     <language>en-US</language>
     <projectUrl>https://go.microsoft.com/fwlink/?linkid=848145</projectUrl>
     <licenseUrl>https://raw.githubusercontent.com/Azure/azure-webjobs-sdk-templates/8ba7a6f1f907edb5de30c1ef4791d97711d84cb6/LICENSE.txt</licenseUrl>
```
Comment (fabiocav) [inline Functions.Templates/ProjectTemplates.nuspec]: Add repository info/properties.

File: appveyor.yml (hunk 1 above)
Comment (soninaren) [reply appveyor.yml]: Needed to generate the correct package format for testing in VS. Going to revert to this before merging.

File: Functions.Templates/ProjectTemplates.nuspec (hunk 3 above)
Comment (soninaren) [reply Functions.Templates/ProjectTemplates.nuspec]: done.

File: Functions.Templates/ProjectTemplates.nuspec (hunk 2 above)
Comment (soninaren) [reply Functions.Templates/ProjectTemplates.nuspec]: done.

File: Functions.Templates/ProjectTemplates.nuspec (hunk 4 above)
Comment (soninaren) [reply Functions.Templates/ProjectTemplates.nuspec]: done
